src/js/StyledIcon.js

```diff
@@ -45,11 +45,11 @@ const StyledIcon = styled(IconInner)`
   flex: 0 0 auto;
 
   ${({ size = 'medium', theme }) => `
-    width: ${theme.icon.size[size] || size};
-    height: ${theme.icon.size[size] || size};
+    width: ${theme.global.icon.size[size] || size};
```

Comment:
can't test this right now but shouldn't it be props.theme....?

Reply:
I don't think so, it's being deconstructed from IconInner on line 47.
also see here https://github.com/grommet/grommet-icons/blob/f3f04ea69062774ef9d7df02c98401131ba97cd9/src/js/StyledIcon.js#L39